mineinabyss-features/src/main/kotlin/com/mineinabyss/features/playerprofile/PlayerProfileMenu.kt

```diff
@@ -195,6 +194,7 @@ fun MittyTokenBalance(player: OfflinePlayer) {
 @Composable
 fun GuildButton(player: OfflinePlayer, viewer: Player) {
     Button(enabled = player.hasGuild() && !viewer.hasGuild(), onClick = {
+        //TODO suspicious call, this seems to od nothing?
         guiy { player.getGuildName()?.let { GuildScreen.GuildLookupMembers(it) } }
```

Comment:
this should open the menu for a guild if the player for the profile has one for the viewer iirc

```kt
guiy(viewer) { GuildScreen.GuildLookupMembers(player.getGuildName()) }
```
or something